src/main/java/core/Duke.java

```diff
@@ -0,0 +1,68 @@
+package core;
+
+import command.Command;
+import dukexception.DukeException;
+import javafx.scene.control.Label;
+
+
```

Comment:
Great abstraction to different classes really makes your Duke neat and only show what's needed!